.github/scripts/build-quickstarts.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash -e
 
-cp -f maven-settings.xml ~/.m2/settings.xml
+if [ -n "$PRODUCT" ] && [ "$PRODUCT" == "true" ]; then
+  args="-s $PRODUCT_MVN_SETTINGS  -Dmaven.repo.local=$PRODUCT_MVN_REPO"
+else
+  args="-s maven-settings.xml"
+fi
```

Comment:
Just a nitpick. I think we could set some `$DIST` var in this IF and use it later when copying the authz-js-policies. That would save us from one more ugly IF. :)